lib/java_buildpack/framework/dynatrace_one_agent.rb

```diff
@@ -68,8 +68,12 @@ def release
 
         manifest = agent_manifest
 
-        @droplet.java_opts.add_agentpath(agent_path(manifest))
-        @droplet.java_opts.add_preformatted_options('-Xshare:off')
+        environment_variables = @droplet.environment_variables
+        environment_variables.add_environment_variable(LD_PRELOAD, agent_path(manifest))
+
+        if enable_fips?
+          File.delete(@droplet.sandbox + 'agent/dt_fips_disabled.flag')
+        end
```

Comment:
```suggestion
        File.delete(@droplet.sandbox + 'agent/dt_fips_disabled.flag') if enable_fips?
```
To satisfy linter complaint